src/mocks/http.php

```diff
@@ -35,22 +40,43 @@ class Http
 	protected static $_default_404_verbal = false;
 	protected static $_instance           = false;
 
+	/**
+	 * Constructor
+	 *
+	 * @codeCoverageIgnore  We can't emulate both WordPress < 6.2 and 6.2+ test environments at the same time.
```

Comment:
There is still a test + assertion for this, but it doesn't handle both conditions.